<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 85, 127);
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="calendar_label">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>21</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Календарь</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 85, 127);
alternate-background-color: rgb(85, 255, 255);
border-top-color: rgb(255, 85, 255);
color: rgb(179, 158, 200);
color: rgb(170, 170, 255);
alternate-background-color: rgb(85, 85, 127);
</string>
       </property>
       <property name="minimumDate">
        <date>
         <year>2000</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
       <property name="maximumDate">
        <date>
         <year>3000</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
       <property name="gridVisible">
        <bool>false</bool>
       </property>
       <property name="horizontalHeaderFormat">
        <enum>QCalendarWidget::ShortDayNames</enum>
       </property>
       <property name="verticalHeaderFormat">
        <enum>QCalendarWidget::NoVerticalHeader</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="diary_btn">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(96, 76, 127);</string>
       </property>
       <property name="text">
        <string>Дневник</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
